### PR TITLE
Removed CDN and added assets folder

### PR DESCRIPTION
**Pull request by sudheshna-donthineni**

### CODE REVIEW

File: src/assets/images/autorenew.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#000000"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M12 6v3l4-4-4-4v3c-4.42 0-8 3.58-8 8 0 1.57.46 3.03 1.24 4.26L6.7 14.8c-.45-.83-.7-1.79-.7-2.8 0-3.31 2.69-6 6-6zm6.76 1.74L17.3 9.2c.44.84.7 1.79.7 2.8 0 3.31-2.69 6-6 6v-3l-4 4 4 4v-3c4.42 0 8-3.58 8-8 0-1.57-.46-3.03-1.24-4.26z"/></svg>
```
Comment (deepansh96) [inline src/assets/images/autorenew.svg]: rename this file to loading_spinner or something

File: src/assets/images/add_circle.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#C62828"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M13 7h-2v4H7v2h4v4h2v-4h4v-2h-4V7zm-1-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/></svg>
```
Comment (deepansh96) [inline src/assets/images/add_circle.svg]: This circle is coming out to be red? Is it on purpose?

Comment (deepansh96) [reply src/assets/images/add_circle.svg]: Screenshot 2021-10-19 at 11 37 52 AM

File: src/assets/images/remove_circle.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#388E3C"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M7 11v2h10v-2H7zm5-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/></svg>
```
Comment (deepansh96) [inline src/assets/images/remove_circle.svg]: And the remove circle icon is coming out to be green. Should be opposite right?

Screenshot 2021-10-19 at 11 38 33 AM

File: src/components/SRNEntry.vue [hunk 4]
```diff
@@ -63,10 +63,9 @@
         @click="addField"
         class="flex flex-row mx-auto p-2 items-center border-2 rounded-xl bg-gray-200 btn"
       >
-        <span class="material-icons text-green-500 text-4xl pr-2">
-          add_circle_outline
-        </span>
-        <div class="border-l-2 border-gray-500 pl-4">
+       <inline-svg class="text-4xl pr-1"
```
Comment (deepansh96) [inline src/components/SRNEntry.vue]: I see that you've changed `pr-2` to `pr-1` but that is just a workaround and this causes the icon to become super small.
Ideally what you want to do, is in the `.svg` file, remove all the `fill`, `height`, `width` tags. You don't want that hardcoded into your svg files. When you remove that, then you can control all those attributes here, where you're using the svg.

File: src/components/SRNEntry.vue [hunk 5]
```diff
@@ -40,10 +40,10 @@
 
       <div class="my-auto px-3" v-show="ifUserEnteredMoreThanOne">
         <button
-          class="material-icons text-red-500"
           @click="removeField(index, userIDList)"
         >
-          remove_circle
+          <inline-svg
+          :src="require('@/assets/images/remove_circle.svg')"></inline-svg>
```
Comment (deepansh96) [inline src/components/SRNEntry.vue:46]: (read this comment after reading the comment attached to add_circle.svg below) 
You'll have to do the same thing here, as discussed in the other comment, remove all the fill, height, width tags from the original svg file and then specify height and width here

**Review by deepansh96 — CHANGES_REQUESTED** [review]

File: src/assets/images/add_circle.svg [hunk 6]
```diff
@@ -1,3 +1,3 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#C62828"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M13 7h-2v4H7v2h4v4h2v-4h4v-2h-4V7zm-1-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/></svg>
+<svg xmlns="http://www.w3.org/2000/svg" ><path d="M0 0h24v24H0V0z" fill="none"/><path d="M13 7h-2v4H7v2h4v4h2v-4h4v-2h-4V7zm-1-5C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/></svg>
```
Comment (deepansh96) [inline src/assets/images/add_circle.svg]: Don't remove viewbox! It's necessary for the content inside an svg to be positioned properly. You can read about what a viewbox is.

File: src/assets/images/loading_spinner.svg [hunk 7]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" ><path d="M0 0h24v24H0V0z" fill="none"/><path d="M12 6v3l4-4-4-4v3c-4.42 0-8 3.58-8 8 0 1.57.46 3.03 1.24 4.26L6.7 14.8c-.45-.83-.7-1.79-.7-2.8 0-3.31 2.69-6 6-6zm6.76 1.74L17.3 9.2c.44.84.7 1.79.7 2.8 0 3.31-2.69 6-6 6v-3l-4 4 4 4v-3c4.42 0 8-3.58 8-8 0-1.57-.46-3.03-1.24-4.26z"/></svg>
```
Comment (deepansh96) [inline src/assets/images/loading_spinner.svg]: You can remove `fill` altogether

Comment (deepansh96) [reply src/assets/images/loading_spinner.svg]: also, don't remove viewbox

File: src/assets/images/remove_circle.svg [hunk 8]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="24px" viewBox="0 0 24 24" width="24px" fill="#388E3C"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M7 11v2h10v-2H7zm5-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg"><path d="M0 0h24v24H0V0z" fill="none"/><path d="M7 11v2h10v-2H7zm5-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"/></svg>
```
Comment (deepansh96) [inline src/assets/images/remove_circle.svg]: - don't remove viewbox
- remove fill

**Review by deepansh96 — CHANGES_REQUESTED** [review]

File: src/assets/images/remove_circle.svg (hunk 8 above)
Comment (deepansh96) [reply src/assets/images/remove_circle.svg]: Also, increase the size of the loading spinner, it's very small

File: src/assets/images/loading_spinner.svg (hunk 7 above)
Comment (sudheshna-donthineni) [reply src/assets/images/loading_spinner.svg]: The fill is the attribute for that specific path. For example, it's a hollow circle. if I remove the fill, it becomes a solid circle.

**Comment by deepansh96** [comment]
There's one bug (as shown below) and that is because you used the same viewBox in all SVGs. Look at the spinner. It's not spinning about it's center.

https://user-images.githubusercontent.com/20169829/138041928-5433991e-b433-44a7-98bb-1d1b2ebb369d.mov

You should use the viewBox value which was originally present in the svg

File: src/components/SRNEntry.vue [hunk 9]
```diff
@@ -63,10 +63,9 @@
         @click="addField"
         class="flex flex-row mx-auto p-2 items-center border-2 rounded-xl bg-gray-200 btn"
       >
-        <span class="material-icons text-green-500 text-4xl pr-2">
-          add_circle_outline
-        </span>
-        <div class="border-l-2 border-gray-500 pl-4">
+       <inline-svg class="fill-current text-green-600 pt-1 pl-1 h-10 w-10"
```
Comment (deepansh96) [inline src/components/SRNEntry.vue]:
```suggestion
       <inline-svg class="fill-current text-green-600 h-10 w-10 pr-1"
```

File: src/components/SRNEntry.vue [hunk 10]
```diff
@@ -2,9 +2,9 @@
   <!-- loading spinner -->
   <div v-if="isLoading" class="h-full w-full fixed z-50">
     <div class="flex mx-auto w-full h-full">
-      <span class="material-icons text-black text-4xl m-auto animate-spin">
-        autorenew
-      </span>
+      <inline-svg class="text-black text-4xl m-auto animate-spin h-10 w-10"
```
Comment (deepansh96) [inline src/components/SRNEntry.vue]:
```suggestion
      <inline-svg class="text-black text-4xl m-auto animate-spin h-20 w-20"
```

File: src/components/SRNEntry.vue [hunk 11]
```diff
@@ -40,10 +40,10 @@
 
       <div class="my-auto px-3" v-show="ifUserEnteredMoreThanOne">
         <button
-          class="material-icons text-red-500"
           @click="removeField(index, userIDList)"
         >
-          remove_circle
+          <inline-svg class="fill-current text-red-600 pt-2 pl-2 h-10 w-10"
```
Comment (deepansh96) [inline src/components/SRNEntry.vue]:
```suggestion
          <inline-svg class="fill-current text-red-600 h-8 w-8"
```

**Review by deepansh96 — CHANGES_REQUESTED** [review]

**Pull request merged**